src/components/__tests__/__snapshots__/Header.spec.js.snap

```diff
@@ -123,30 +123,89 @@ c0,0.1,0.1,0.2,0.1,0.4L115.4,16.3L115.4,16.3z"
               </svg>
             </div>
             <div
-              class="zep-grid__col zep-grid__col--align-self-center zep-grid__col--xxs-1-4 zep-grid__col--xs-1-6 zep-grid__col--m-2-8"
+              class="zep-grid__col zep-grid__col--align-self-center zep-grid__col--xxs-0-4  zep-grid__col--m-6-8 Header-searchLarge-0-1-5"
```

Comment:
.zep-grid__col--xxs-0-4 is not possible only 1 - 4 of 4 and you have to include col--xs as well between xxs and m